### PR TITLE
Board size in config

### PR DESCRIPTION
**Pull request by mcroitor**
Please, accept `gui.py` modification for  fontSize value reading from main.yaml

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,3 +9,4 @@ resources.py
 olive.log
 olive.spec
 test.json
+nbproject/
```
Comment (dturevski) [inline .gitignore:12]: Please remove the IDE artifacts from this pull request

File: conf/chest.yaml [hunk 2]
```diff
@@ -1 +1,2 @@
-{options: -r -LS -M 128, path: WinChest.exe}
+options: -r -LS -M 128
+path: WinChest.exe
```
Comment (dturevski) [inline conf/chest.yaml:2]: YAML reformatted, please revert, this will only produce noise in the source log.

File: conf/lang.yaml [hunk 3]
```diff
@@ -285,6 +285,11 @@ MI_Export_Image:
   rs: Дијаграм у имиџ фајл
   ru: Диаграмму в файл изображения
   de: Diagram als Bilddatei exportieren
+MI_Export_RTF:
```
Comment (dturevski) [inline conf/lang.yaml:288]: Where is the RTF export support? If it is not part of this PR, these changes should be removed.

File: conf/main.yaml [hunk 4]
```diff
@@ -1,27 +1,88 @@
...
+  option: MoveNumbers
+- enabled: false
+  icon: contrast.svg
+  option: WhiteToPlay
+- enabled: false
+  icon: checklist.svg
+  option: NoShortVariations
+- enabled: false
+  icon: no-bombs.svg
+  option: NoThreat
```
Comment (dturevski) [inline conf/main.yaml:87]: diff noise

File: nbproject/project.properties [hunk 5]
```diff
@@ -0,0 +1,5 @@
+java.lib.path=
+main.file=olive.py
+olive-gui.dir=.
+platform.active=Python_2.7.10
+python.lib.path=
```
Comment (dturevski) [inline nbproject/project.properties:5]: IDE artifacts

File: nbproject/project.xml [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+    <configuration>
+        <data xmlns="http://nbpython.dev.java.net/ns/php-project/1">
+            <name>olive-gui</name>
+            <sources>
+                <root id="olive-gui.dir"/>
+            </sources>
+            <tests/>
+        </data>
+    </configuration>
+</project>
```
Comment (dturevski) [inline nbproject/project.xml:13]: IDE artifacts

File: nbproject/project.properties (hunk 5 above)
Comment (dturevski) [reply nbproject/project.properties:5]: But these are under the .gitignore?

File: .gitignore (hunk 1 above)
Comment (mcroitor) [reply .gitignore:12]: accept

File: conf/lang.yaml (hunk 3 above)
Comment (mcroitor) [reply conf/lang.yaml:288]: i removed it, it not finalised.

File: .gitignore (hunk 1 above)
Comment (mcroitor) [reply .gitignore]: solved

File: conf/main.yaml (hunk 4 above)
Comment (mcroitor) [reply conf/main.yaml]: solved

File: nbproject/project.properties (hunk 5 above)
Comment (mcroitor) [reply nbproject/project.properties]: .gitignore was updated to late

File: nbproject/project.xml (hunk 6 above)
Comment (mcroitor) [reply nbproject/project.xml]: solved

File: conf/chest.yaml (hunk 2 above)
Comment (mcroitor) [reply conf/chest.yaml]: solved
